src/handler.ts

```diff
@@ -9,11 +10,11 @@ import logger from './util/logger';
  * @returns {Promise<string>}
  */
 export const handler = async (
-  event: EventBridgeEvent<any, any>,
+  event: EventBridgeEvent<'VT Booking', VtBooking>,
 ): Promise<string> => {
   logger.debug(`event: ${JSON.stringify(event, null, 2)}`);
 
-  await insertVtBooking();
+  await insertVtBooking(event.detail);
```

Comment:
are we doing any validation on `event.details`? maybe add basic validation as starter like required for example.

Reply:
Done.